Improved Group Membership Extension section (#75)
Haivision/srt-rfc: draft-sharabayko-mops-srt.md

#### Group Membership Extension

> The Group Membership handshake extension is reserved for the future and is
> going to be used to distinguish single SRT connections

Reviewer: ```suggestion
The Group Membership handshake extension is reserved for the future and is
going to be used to allow multipath SRT connections.
```

GroupID: 32 bits.
: The identifier of a group whose members include the sender socket that is making a connection.
  The target socket that is interpreting GroupID should belong to the corresponding group on a target side

>   or create one if the group does not exist.

Reviewer 2: ```
The target socket that is interpreting GroupID should belong to the corresponding group on the target side. If 
such a group does not exist, the target socket should create it.
```

>   The target socket that is interpreting GroupID should belong to the corresponding group on the target side.
>   If such a group does not exist, the target socket should create it.

Reviewer: "Should" -> "must" because it must belong to a group, or consider creating one. Although not 100% sure about the word "must" here, so upon your decision. 🤔 
"Should" -> "may" because it depends e.g. on whether a socket is allowed to accept group connections or not.

```suggestion
  The target socket that is interpreting GroupID must belong to the corresponding group on the target side.
  If such a group does not exist, the target socket may create one.
```
